pkg/server/handler/v1alpha1/workflowrun.go

```diff
@@ -459,3 +466,150 @@ func GetContainerLogs(ctx context.Context, project, workflow, workflowrun, tenan
 
 	return folderReader, headers, nil
 }
+
+// ReceiveStageArtifacts receives artifacts produced by workflowrun stage.
+func ReceiveStageArtifacts(ctx context.Context, workflowrun, namespace, stage string) error {
```

Comment:
Keep consistent name style with other functions?

```suggestion
func ReceiveArtifacts(ctx context.Context, workflowrun, namespace, stage string) error {
```